plugins/weather.py

```diff
@@ -0,0 +1,23 @@
+import urllib
+import json
+import ircbot.plugin
+
+class WeatherPlugin(ircbot.plugin.Plugin):
+	@ircbot.plugin.command('weather')
+	def weather(self, cmd):
+		if len(cmd.args) < 2:
+			return 'Usage: !weather city state/country'
+		try:
+			url = 'http://api.wunderground.com/api/ba6f8fecb3824c22/geolookup/conditions/q/' + cmd.args[1] + '/' + cmd.args[0] + '.json'
```

Comment:
Change this to use str.format as well. Also see comment about how to get the wunderground API key from config.